app/modules/entities/components/layouts/entity_list_row.svelte

```diff
@@ -0,0 +1,157 @@
+<script>
+  import { isSubEntitiesType } from '#entities/components/lib/works_browser_helpers'
+  import { omitClaims } from '#entities/components/lib/work_helpers'
+  import { loadInternalLink } from '#lib/utils'
+  import ImagesCollage from '#components/images_collage.svelte'
+  import Infobox from './infobox.svelte'
+  import { getContext } from 'svelte'
+
+  export let entity,
+    relatedEntities,
+    showInfobox = true,
+    listDisplay = false,
+    displayUri
+
+  let { claims, label, image, images, pathname, serieOrdinal, subtitle, title, type, uri } = entity
+
+  const mainPropertyType = {
+    author: 'wdt:P50',
+    publisher: 'wdt:P123',
+    genre: 'wdt:P136',
+    series: 'wdt:P179',
+    collection: 'wdt:P195',
+    edition: 'wdt:P629',
+    translator: 'wdt:P655',
+    subject: 'wdt:P921',
+  }
+
+  const layoutContext = getContext('layout-context')
+
+  const prop = mainPropertyType[layoutContext]
+  const hasOnlyOneClaimValue = prop => claims?.[prop]?.length === 1
+  if (prop && hasOnlyOneClaimValue(prop)) {
+    claims = omitClaims(claims, [ prop ])
+  }
+</script>
+<div class="entity-wrapper">
+  <div
+    class="cover"
+    class:hasSubEntities={isSubEntitiesType(type)}
+  >
+    <a
+      href={pathname}
+      on:click={loadInternalLink}
+      {title}
+      tabindex="-1"
```

Comment:
why the need to mess with the tabindex?

Reply:
cant remember or reproduce, deleted in 35d304200